src/Attributes/AdditionalProperties.php

```diff
@@ -98,6 +99,7 @@ public function __construct(
             'allOf' => $allOf ?? Generator::UNDEFINED,
             'anyOf' => $anyOf ?? Generator::UNDEFINED,
             'oneOf' => $oneOf ?? Generator::UNDEFINED,
+            'additionalProperties' => $additionalProperties ?? Generator::UNDEFINED,
```

Comment:
Have you tested this? All other nested objects are passed into the parent in this line:
```
            'value' => $this->combine($items, $discriminator, $externalDocs, $attachables),
```

Reply:
For the value of additionalProperties i just copied the "OAT\Schema" style. 
I have changed it to an additional argument of the `combine()` method and the endresult is the same.
Aka it generates the same YAML file.

Change is on its way